x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataTypes.java

```diff
@@ -67,6 +69,9 @@ public static DataType fromJava(Object value) {
         if (value instanceof Short) {
             return SHORT;
         }
+        if (value instanceof LocalDateTime) {
```

Comment:
Do we create a `LocalDateTime`  instead of `ZonedDateTime` anywhere?

Reply:
Nope, that's a leftover, thx.